fixtures/stairville/revueled-120-cob-true-white.json

```diff
@@ -49,24 +46,24 @@
         }
       ]
     },
-    "Intensity Warm White": {
+    "Warm White": {
+      "defaultValue": 0,
       "capability": {
-        "type": "ColorIntensity",
-        "color": "Warm White"
+        "type": "Intensity"
       }
     },
-    "Intensity Cold White": {
+    "Cold White": {
+      "defaultValue": 0,
       "capability": {
         "type": "ColorIntensity",
-        "color": "Cold White"
+        "color": "White"
       }
     },
     "Strobe": {
       "capabilities": [
         {
           "dmxRange": [0, 5],
-          "type": "ShutterStrobe",
-          "shutterEffect": "Open"
+          "type": "NoFunction"
         },
         {
           "dmxRange": [6, 255],
```

Comment:
The user manual is funny.  It says 0-5 is "Strobe on", and 6-255 is "Strobe speed, slow to fast (1 Hz - 30 Hz)", which doesn't really make sense.

My guess is that they mean:

```
  {
    "dmxRange": [0, 5],
    "type": "ShutterStrobe", 
    "shutterEffect": "Open"`
  },
  {
    "dmxRange": [6, 255],
    "type": "ShutterStrobe",
    "shutterEffect": "Strobe",
    "speedStart"="slow",
    "speedEnd"="fast"
  }
```

Reply:
I've fixed the `StrobeSpeed` capability in #3764. The other changes look only cosmetic, so this PR can be closed. Thanks for checking, @kengruven!